lib/questions.rb

```diff
@@ -63,33 +82,48 @@ def separate_array_into_even_and_odd_numbers(array)
 # e.g. 'bob'. So in the array ['bob', 'radar', 'eat'], there
 # are 2 palindromes (bob and radar), so the method should return 2
 def number_of_elements_that_are_palindromes(array)
+  counter = 0
+  array.each do |word|
+    word == word.split(//).reverse.join ? counter += 1 : counter
+  end
+  return counter
 end
 
 # return the shortest word in an array
 def shortest_word_in_array(array)
+  array.sort! {|a,b| a.length <=> b.length}
+  array.shift
 end
 
-# return the shortest word in an array
+# return the longest word in an array
 def longest_word_in_array(array)
+  array.sort! {|a,b| a.length <=> b.length}
```

Comment:
Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.